google-cloud-bigquery/pom.xml

```diff
@@ -12,7 +12,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-pom</artifactId>
-    <version>0.8.1-alpha</version>
+    <version>0.8.2-alpha-SNAPSHOT</version>
```

Comment:
This looks so wrong.  SNAPSHOT is by it's very nature not released.  I kind of doubt you'll get the right result with this, as it's not properly defined in the format from the manual.

I would suggest the following `<version>0.8.2-SNAPSHOT</version><!-- alpha -->`

@tswast Thoughts?